v2/src/log_sink_console.c

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdio.h>
+
+#include "macro_utils/macro_utils.h"
+
+#include "c_logging/logger.h"
+#include "c_logging/log_context.h"
+
+#include "c_logging/log_sink_console.h"
+
+void log_sink_console_log(LOG_LEVEL log_level, LOG_CONTEXT_HANDLE log_context, const char* message, const char* file, const char* func, int line)
+{
+    time_t t = time(NULL);
+
+    (void)printf("%s %s Time:%.24s File:%s Func:%s Line:%d %s\r\n", MU_ENUM_TO_STRING(LOG_LEVEL, log_level), (log_context != NULL) ? log_context->context_string : "", ctime(&t), file, func, line, message);
+}
+
+const LOG_SINK console_log_sink = { log_sink_console_log };
```

Comment:
>log_sink_console_log [](http://example.com/codeflow?start=36&length=20)

I've come to REALLY like lately designated initializers, so consider adding those :)

Reply:
Done